core/lowering/passes/exception_elimination.cpp

```diff
@@ -1,3 +1,4 @@
+#include <torch/csrc/jit/passes/constant_pooling.h>
```

Comment:
Hey! Sorry to barge in, and thank you for handling this issue! (which I also encountered and found my way here after analyzing gdb stack trace)
Just a nit here, these should be probably changed to quotes.